src/index.ts

```diff
@@ -15,6 +15,7 @@ export { default as Label } from "./components/Label";
 export { default as Link } from "./components/Link";
 export { default as List } from "./components/List";
 export { default as Loader } from "./components/Loader";
+export { default as ShowMore } from "./components/SummaryButton";
```

Comment:
Not sure if you meant to leave this in.

Reply:
Ah, good catch. I guess VSCode renamed the import when I renamed the file, but it didn't rename the export name.